stats.php

```diff
@@ -72,19 +72,27 @@ function getPriceConverted($price, $currency, $database) {
 
 
 // Calculate active subscriptions
-$query = "SELECT COUNT(*) AS active_subscriptions FROM subscriptions";
+$query = "SELECT COUNT(*) AS active_subscriptions FROM subscriptions WHERE inactive = 0";
 $stmt = $db->prepare($query);
 $stmt->bindParam(':criteria', $criteria, SQLITE3_INTEGER);
 $result = $stmt->execute();
 $row = $result->fetchArray(SQLITE3_ASSOC);
 $activeSubscriptions = $row['active_subscriptions'];
 
+// Calculate inactive subscriptions
+$query = "SELECT COUNT(*) AS inactive_subscriptions FROM subscriptions WHERE inactive = 1";
+$stmt = $db->prepare($query);
+$stmt->bindParam(':inactive', $inactive, SQLITE3_INTEGER);
+$result = $stmt->execute();
+$row = $result->fetchArray(SQLITE3_ASSOC);
+$inactiveSubscriptions = $row['inactive_subscriptions'];
+
 // Calculate total monthly price
 $mostExpensiveSubscription = 0;
 $amountDueThisMonth = 0;
 $totalCostPerMonth = 0;
 
-$query = "SELECT name, price, frequency, cycle, currency_id, next_payment, payer_user_id, category_id FROM subscriptions";
+$query = "SELECT name, price, frequency, cycle, currency_id, next_payment, payer_user_id, category_id FROM subscriptions WHERE inactive = 0";

```

Comment:
This query was update meanwhile and there's now a conflict. Can you please add `payment_method_id` ? 

`$query = "SELECT name, price, frequency, cycle, currency_id, next_payment, payer_user_id, category_id FROM subscriptions WHERE inactive = 0";`

Reply:
updated to 
``` php
$query = "SELECT name, price, frequency, cycle, currency_id, next_payment, payer_user_id, category_id, payment_method_id FROM subscriptions WHERE inactive = 0";
```